src/modules/oauth/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import reducer from './services/reducer'
+
+import Module from '../../abc/Module';
+import {HEADER_POS_RIGHT} from '../../services/HeaderService';
+
+import UserDropdown from './containers/UserDropdown';
+import RegisterContainer from './containers/RegisterContainer'
+import AuthContainer from './containers/AuthContainer'
+import TeskalabsAuth from './containers/TeskalabsAuth'
+import GitHubAuth from './containers/GitHubAuth'
+import AuthService from './services/AuthService'
+
+
+
+export default class AuthenticationModule extends Module {
+    constructor(app, name){
+        super(app, "Authentication");
+
```

Comment:
Meh ... don't hardcode that vendor-specific things (TL Auth, GitHub) ... it shouldn't be part of the library.
On the other hand, it should be captured in examples/docs/